src/Lykke.Service.LiteCoin.API/Controllers/OperationsController.cs

```diff
@@ -93,20 +95,24 @@ public async Task<BuildTransactionResponse> BuildSingle([FromBody] BuildSingleTr
 
                 fromAddressPubkey = _addressValidator.GetPubkey(pubKeyString);
             }
+            if (await _operationEventRepository.Exist(request.OperationId, OperationEventType.Broadcasted))
```

Comment:
What if Broadcasted event will be added to the repo right after this condition by another thread?

Reply:
Looks like we could have a problem here, since two different repositories are used for Broadcasted events and built transactions